string value
string[] list
string client_name
string group_name
string data_name
---
string resp
